internal/wasip1/errno.go

```diff
@@ -263,47 +264,47 @@ var errnoToString = [...]string{
 // Note: Coercion isn't centralized in sys.FSContext because ABI use different
 // error codes. For example, wasi-filesystem and GOOS=js don't map to these
 // Errno.
-func ToErrno(errno syscall.Errno) Errno {
+func ToErrno(errno sys.Errno) Errno {
```

Comment:
This file as all the errnos we currently use. This plus the ones in GOOS=js were the ones defined so far.